pages/signinLinkSent.js

```diff
@@ -37,24 +31,22 @@ class SignInLinkSent extends Component {
     const { email } = this.props;
     const isValidEmail = email && isEmail(email);
     return (
-      <Page title="Login Link Sent" noRobots>
-        <Container pt={[4, 5]} pb={6} px={3} background="linear-gradient(180deg, #EBF4FF, #FFFFFF)" textAlign="center">
-          <Flex justifyContent="center" mb={4}>
-            <Icon size="60" />
-          </Flex>
-          <H3 as="h1" fontWeight="bold">
+      <Page title="Login Link Sent" noRobots showFooter={false}>
+        <Container pt={[4, 5]} pb={6} px={3} textAlign="center">
+          <Image src="/static/images/sign-in-illustration.png" width="624px" height="372px" />
```

Comment:
Did you download this image with a x2 resolution? I'm not sure if it's retina-ready

Reply:
Ops, you are right. 🤯 I've updated the resolution. Should be good now.